coverbot/cover_updater.py

```diff
@@ -22,7 +22,7 @@
                 olid = _json["key"].split("/")[-1]
                 edition_obj = ol.Edition.get(olid)
                 if not len(getattr(edition_obj, "covers", [])):
-                    fout.write(edition_obj.olid + "\n")
+                    fout.write("".join([edition_obj.olid, "\n"]).encode())
```

Comment:
```suggestion
                    fout.write(f"{edition_obj.olid}\n".encode())
```
Python f-strings are shorter, faster, and easier to read.  https://www.scivision.dev/python-f-string-speed